mumerge.py

```diff
@@ -211,6 +221,8 @@ def inputs_processor():
                     sampid_col = i
                 elif col == 'group':
                     group_col = i
+                elif col == 'bam':
```

Comment:
I don't see any record of this being used below. Am I missing something? If this isn't being used can you remove it?